src/main.dart

```diff
@@ -1,8 +1,129 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:io';
+import 'task.dart';
+import 'taskList.dart';
+
+//you can make it global so you can access it from anywhere
+List<Task> _tasks = [];
+TaskList tasks = TaskList([]);
+
+void main(List<String> arg) {
+//Listing Choices
+  void showTasks() {
+    print("This is your Task List app");
+    print("**************************");
+    print("What Do You Need?");
+    print("1.Add Task");
+    print("2.List Task");
+    print("3.Update A Task");
+    print("4.Delete A Task");
+    print("5.Exit");
+  }
+
+//Making Choice
+  int choice = 0;
+  while (choice != 5) {
+    showTasks();
+    bool validChoice = false;
+    while (!validChoice) {
+      try {
+        stdout.write("Your Choice?");
```

Comment:
could make this prettier
```suggestion
        stdout.write("Your Choice: ");
```